src/tag_index.c

```diff
@@ -226,12 +226,12 @@ static void TagReader_OnReopen(void *privdata) {
       InvertedIndex *idx = TagIndex_OpenIndex(ctx->idx, ir->record->term.term->str,
                                               ir->record->term.term->len, DONT_CREATE_INDEX, &sz);
       if (idx == TRIEMAP_NOTFOUND || ir->idx != idx) {
-        // the inverted index was collected entirely by GC, lets stop searching.
+        // the inverted index was collected entirely by GC.
         // notice, it might be that a new inverted index was created, we will not
```

Comment:
I think the comment should be updated a bit further.
Especially since the code is doing something opposite to what the comment is saying.

Reply:
The comment is talking about the `ir->idx != idx` condition, which is still valid (although not possible since the fields were moved from the key space)